src/pages/Login/Login.js

```diff
@@ -1,7 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import Sns from './Sns';
+
 const Login = () => {
-  return <div />;
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setPerson({ ...person, [name]: value });
+  };
+  // 아이디, 비밀번호 정규식
+  const passwordRegExp =
+    /^(?=.*[a-z])(?=.*\d)(?=.*[$@$!%*#?&])[a-z\d$@$!%*#?&]{8,16}$/;
+  const idRegExp = /^[a-z]+[a-z0-9]{4,12}$/;
+
+  const validation = idRegExp.test(person.id) && passwordRegExp.test(person.pw);
```

Comment:
boolean naming convention에 따라 is를 붙여주세요!
```suggestion
  const isAllValid = idRegExp.test(person.id) && passwordRegExp.test(person.pw);
```